src/svg/ds4/icons.svg

```diff
@@ -146,6 +155,9 @@
     <symbol id="icon-information" viewBox="0 0 24 24">
         <path fill="#0654ba" d="M12.007 9.753a.849.849 0 0 0-.849.849v6.553a.85.85 0 0 0 1.683.004v-6.557a.849.849 0 0 0-.834-.849zm1.057-2.88a1.016 1.016 0 1 1-2.032 0 1.016 1.016 0 0 1 2.032 0zm-1.057 17.141C5.376 24.014 0 18.638 0 12.007 0 5.376 5.376 0 12.007 0c6.631 0 12.007 5.376 12.007 12.007v.015c-.008 6.625-5.381 11.992-12.007 11.992zm0-22.261c-5.671 0-10.268 4.597-10.268 10.268s4.597 10.268 10.268 10.268 10.268-4.597 10.268-10.268S17.678 1.753 12.007 1.753z"/>
     </symbol>
+    <symbol id="icon-information-small" viewBox="0 0 24 24">
```

Comment:
In order for this to be in sync with my version call this `icon-information-filled-small`

Reply:
As discussed offline, I won't update this since for DS4 it's only used in docs and demos on skin or ebayui.